### DISCUSSION TITLE
creating a BOTB from BOTA in same environment

### DISCUSSION THREAD

**Discussion by dacostadaniel** [microsoft/powerplatform-build-tools#945] [Q&A]
Hello Colleagues

Can anyone help with creating a BOTB from BOTA in same environment please?

**pac copilot extract-template --environment 1e33b310-6f9d-e3ff-8941-a0862324b21e --bot 674457c4-e1e5-ee11-904c-0022480ac6c9 --templateFileName CALINT.yaml**

the above pac copilot extract-template works however,

the above pac copilot create fails

**pac virtual-agent create --SchemaName cr7d0_calint --TemplateFileName CALINTER.yaml --environment 1e33b310-6f9d-e3ff-8941-a0862324b21e --displayName CALINTER --solution CALINTER**

Loaded 357 components for bot 'CALINT' with id a46dccfb-fb85-4479-bff2-f85a22a31b53. Primary language: English, supported languages:
Sorry, the app encountered a non recoverable error and will need to terminate. The exception details have been captured and will be forwarded to the development team, if telemetry has been enabled. Exception Id: 34c67bca-e784-4022-bf79-4a135422a3e9, Exception Type: Microsoft.Bot.Platform.Content.Exceptions.DataverseBadRequestException

any help appreciated.

I just need a copy so I can make minor changes to BOTB rather than kill myself building a new one. 

regards
Daniel da Costa
+447902596815
daniel@educlaas.com

**Comment by haack42** [comment]
Hello Daniel,

The exception contains _"It is invalid to create component bot with the same export key value(s) as an existing component. Please change the key. The current value(s) are schemaname: daniel_calint"_

You can also run `pac telemetry status` to get the path to the log file which should contain this exception contents. For me this showed this message when I tried to use a schemaName already in use:

_2024-06-28 06:24:01.9153 | FTL | bolt.Session :: HTTP Status:BadRequest, ServiceRequestId: 5e0d57d9-ab6c-480a-a031-8b4945843fd4 [0x80072013:ExportKeyInvalidCreate] It is invalid to create component bot with the same export key value(s) as an existing component. Please change the key. The current value(s) are schemaname: crd15_translationTestCopilot ..._

Regards,
Matthias

**Reply by dacostadaniel** [reply]
Hello Sir

Still do not get the results
Please see ...

STEP 1
C:\Users\Sevara Kenjaeva>pac auth who
Connected as [daniel@educlaas.com](mailto:daniel@educlaas.com)

Name:                       SANDBOX
Type:                       User
Cloud:                      Public
Tenant Id:                  0157f289-8461-4324-a135-ef240b05b11d
Tenant Country:             SG
User:                       [daniel@educlaas.com](mailto:daniel@educlaas.com)
Entra ID Object Id:         ad5c86fb-ebfa-46dd-9261-c4cce15af90b
PUID:                       100320027C79DDEA
Token Expires:              26/06/2024 20:51:18 +00:00
Authority:                  https://login.microsoftonline.com/0157f289-8461-4324-a135-ef240b05b11d/
Environment Geo:            NA
Environment Id:             1e33b310-6f9d-e3ff-8941-a0862324b21e
Environment Type:           Regular
Organization Id:            b4b0bcee-5a71-ee11-8bc5-6045bd003027
Organization Unique Name:   unqb4b0bcee5a71ee118bc56045bd003
Organization Friendly Name: eduCLaaS Global
STEP 2
C:\Users\Sevara Kenjaeva>pac auth list
Index Active Kind      Name       User                Cloud  Type Environment                      Environment Url
[1]          UNIVERSAL            [daniel@educlaas.com](mailto:daniel@educlaas.com) Public User Lithan Academy Pte Ltd (default) https://orgff055ba5.crm5.dynamics.com/
[2]   *      UNIVERSAL SANDBOX    [daniel@educlaas.com](mailto:daniel@educlaas.com) Public User eduCLaaS Global                  https://org978b048d.crm.dynamics.com/
[3]          UNIVERSAL PRODUCTION [daniel@educlaas.com](mailto:daniel@educlaas.com) Public User eduCLaaS Global                  https://orge762233f.crm.dynamics.com/


STEP 2
C:\Users\Sevara Kenjaeva>pac copilot list
Connected as [daniel@educlaas.com](mailto:daniel@educlaas.com)
Connected to... eduCLaaS Global
Name                          Bot ID                               Component State Is Managed Solution ID                          Status Code State Code
Lifelong CLaaS | Adapt Card 1 8103306c-774d-4840-a8e9-ae2ee03307c6 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
PohHeng_DemoBot               296f3f00-dee4-ee11-904c-0022480ac6c9 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
HR_MCBot                      574c22c2-87e1-ee11-904c-0022480ac6c9 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
Contoso Customer Service      5727514c-b0f2-ee11-904b-0022480ac6c9 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
Copilot 2                     bba82952-57f2-ee11-904b-0022480ac6c9 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
Callie                        0c5721f5-0da9-ee11-be37-00224809ce3e Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
INT CAL                       84891b31-d72e-ef11-840a-00224802a607 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
Lifelong Learning | CALA STD  d4bd403e-93eb-ee11-a203-000d3a339dfc Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
CAL                           674457c4-e1e5-ee11-904c-0022480ac6c9 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
CALSANDBOX                    c618bc76-ba0d-ef11-9f89-000d3a59ff6a Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
LLL CLaaS┬« | Vertical 1       9e9a1359-e47f-4656-a027-c13826a5d453 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
GAI Module Show Tell          9418e3eb-d6f1-ee11-904b-000d3a339dfc Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
Daniel Today 1200124          91c82035-51df-44db-9e5d-c3f08c6ea6d3 Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned
CALSandbox-v1                 895ef131-cc0d-ef11-9f89-000d3a59ff6a Published       Unmanaged  fd140aae-4df4-11dd-bd17-0019b9312238 Active      Provisioned

STEP 3
C:\Users\Sevara Kenjaeva>pac copilot extract-template --environment 1e33b310-6f9d-e3ff-8941-a0862324b21e --bot 674457c4-e1e5-ee11-904c-0022480ac6c9 --templateFileName NewTestBot.yaml -o
Connected as [daniel@educlaas.com](mailto:daniel@educlaas.com)

Loaded 357 components for bot 'CAL' with id 674457c4-e1e5-ee11-904c-0022480ac6c9. Primary language: English, supported languages:

CAL -> C:\Users\Sevara Kenjaeva\NewTestBot.yaml
C:\Users\Sevara Kenjaeva>pac copilot extract-template --environment 1e33b310-6f9d-e3ff-8941-a0862324b21e --bot 674457c4-e1e5-ee11-904c-0022480ac6c9 --templateFileName NewTestBot.yaml
Connected as [daniel@educlaas.com](mailto:daniel@educlaas.com)

Loaded 357 components for bot 'CAL' with id 674457c4-e1e5-ee11-904c-0022480ac6c9. Primary language: English, supported languages: 

STEP 4
Directory of C:\Users\Sevara Kenjaeva

30/06/2024  20:11    <DIR>          .
30/06/2024  20:11    <DIR>          ..
24/06/2024  10:09    <DIR>          .nuget
16/11/2021  16:09    <DIR>          3D Objects
26/06/2024  23:00         2,979,039 CALDaniel.zip
27/06/2024  15:52         2,138,244 CALINT.yaml
26/06/2024  23:22         2,980,271 CALINT.zip
16/11/2021  16:09    <DIR>          Contacts
14/06/2024  21:58    <DIR>          Desktop
12/06/2024  19:16    <DIR>          Documents
24/06/2024  16:06    <DIR>          Downloads
16/11/2021  16:09    <DIR>          Favorites
11/01/2024  20:51    <DIR>          Intel
16/11/2021  16:09    <DIR>          Links
10/09/2023  19:35    <DIR>          Microsoft
04/05/2024  16:42    <DIR>          Music
30/06/2024  20:11         2,138,244 NewTestBot.yaml
30/06/2024  20:01    <DIR>          OneDrive
26/06/2024  23:59                 0 pac
01/04/2024  18:35    <DIR>          Pictures
16/11/2021  16:09    <DIR>          Saved Games
24/06/2024  15:01    <DIR>          Searches
29/12/2023  17:43                 0 Sti_Trace.log
24/01/2024  19:41    <DIR>          Videos
               6 File(s)     10,235,798 bytes
              18 Dir(s)  67,668,840,448 bytes free

STEP 5 create a solution

 
STEP 6 verify solution created 
Step 7
C:\Users\Sevara Kenjaeva>pac virtual-agent create --schemaName cr7d0_NewTestBot --templateFileName NewTestBot.yaml --environment 1e33b310-6f9d-e3ff-8941-a0862324b21e --displayName NewTestBot --solution NewTestBotSol
Connected as [daniel@educlaas.com](mailto:daniel@educlaas.com)

Loaded 357 components for bot 'NewTestBot' with id 6dcfe561-74e3-4b36-8f98-690ebf3fd45c. Primary language: English, supported languages:
Sorry, the app encountered a non recoverable error and will need to terminate. The exception details have been captured and will be forwarded to the development team, if telemetry has been enabled. Exception Id: c42b1b4e-f967-4e0a-81ec-1c224692dea5, Exception Type: Microsoft.Bot.Platform.Content.Exceptions.DataverseBadRequestException
The diagnostic logs can be found at: C:\Users\Sevara Kenjaeva\AppData\Local\Microsoft\PowerAppsCLI\Microsoft.PowerApps.CLI.1.32.8\tools\logs\pac-log.txt


Files attached Yaml and Log files attached.

 
Regards
Daniel
[NewTestBotyaml.txt](https://github.com/user-attachments/files/16045976/NewTestBotyaml.txt)
[pac-log.txt](https://github.com/user-attachments/files/16045977/pac-log.txt)
[pac-log.txt](https://github.com/user-attachments/files/16045982/pac-log.txt)
[NewTestBotyaml.txt](https://github.com/user-attachments/files/16045983/NewTestBotyaml.txt)

**Reply by haack42** [reply]
Hello,

It seems the error in the log changed from: 
_"It is invalid to create component bot with the same export key value(s) as an existing component. Please change the key. The current value(s) are schemaname: cr7d0_calInt"_

Into this one: 
_"2 is not a valid status code for state code botcomponentState.Active on botcomponent with Id 245f3392-c9d4-4de4-a34f-11516d0ec729."_

There seems to be an issue for import when a copilot has **Inactive** topics. I created a ticket to fix this issue.

In the meantime, as a workaround, you could try to mark the topics all active (e.g. in the template YAML directly) and then deactivate them after the new copilot was created.

Regards,
Matthias

**Reply by dacostadaniel** [reply]
Hello Sir

I tried your work around

So I edited the yaml file to replace Inactive with Active/ so the file.
[NewTestBotActiveyyaml.txt](https://github.com/user-attachments/files/16085366/NewTestBotActiveyyaml.txt)

the extension is txt so I can upload it for you to take a look.

C:\Users\Sevara Kenjaeva>pac virtual-agent create --schemaName cr7d0_NewTestBot --templateFileName NewTestBotActive.yaml --environment 1e33b310-6f9d-e3ff-8941-a0862324b21e --displayName NewTestBot --solution NewTestBotSol
Connected as daniel@educlaas.com

Loaded 357 components for bot 'NewTestBot' with id 771bd830-fb50-4caf-bf8d-892e4f690567. Primary language: English, supported languages:
Sorry, the app encountered a non recoverable error and will need to terminate. The exception details have been captured and will be forwarded to the development team, if telemetry has been enabled. Exception Id: 537644b0-11c2-4058-9bb5-13cbd42e6e0e, Exception Type: Microsoft.Bot.Platform.Content.Exceptions.DataverseBadRequestException
The diagnostic logs can be found at: C:\Users\Sevara Kenjaeva\AppData\Local\Microsoft\PowerAppsCLI\Microsoft.PowerApps.CLI.1.32.8\tools\logs\pac-log.txt

the log file is here.



[pac-log.txt](https://github.com/user-attachments/files/16085392/pac-log.txt)

Is there any chance of we connecting with MS teams (daniel@educlaas.com) to go through the issue please. I am based in europe even though the company is based in Singapore so i am within your time zone.

the log errors

line 49 2024-06-24 09:19:38.8837 | ERR | Microsoft.Identity.Client.IPublicClientApplication :: True MSAL 4.59.0.0 MSAL.Desktop 4.8 or later Windows 10 Home [2024-06-24 09:19:38Z - 1d313150-d94c-4bc1-8be4-55d5857c7f5d] MSAL.Desktop.4.59.0.0.MsalClientException: 
	ErrorCode: authentication_canceled
line 70 2024-06-24 09:19:38.8837 | ERR | bolt.authentication.store.AuthTokenStore :: Exception in CreateAuthProfile()|EXCEPTION: MSAL.Desktop.4.59.0.0.MsalClientException: 
	ErrorCode: authentication_canceled

line 2409 2024-06-26 19:25:38.3097 | ERR | Microsoft.Identity.Client.IPublicClientApplication :: True MSAL 4.59.0.0 MSAL.Desktop 4.8 or later Windows 10 Home [2024-06-26 19:25:38Z - ae2430a1-d159-4537-806a-3755e3e47af5] MSAL.Desktop.4.59.0.0.MsalClientException: 
	ErrorCode: authentication_canceled
line 2430 2024-06-26 19:25:38.3397 | ERR | bolt.authentication.store.AuthTokenStore :: Exception in CreateAuthProfile()|EXCEPTION: MSAL.Desktop.4.59.0.0.MsalClientException: 
	ErrorCode: authentication_canceled
line 2882 Error: Message: Solution manifest import: FAILURE: The following solution cannot be imported: CALSingapore. Some dependencies are missing. The missing dependencies are : <MissingDependencies><MissingDependency><Required type="connectionreference" displayName="cr7d0_sharedregexmatchingip_cd463" solution="Active" id.connectionreferencelogicalname="cr7d0_sharedregexmatchingip_cd463" /><Dependent type="29" displayName="Step X - Email Matching - Mitanshu Garg" id="{c51507f9-350d-ef11-9f89-000d3a59ff6a}" /></MissingDependency></MissingDependencies> , ProductUpdatesOnly : False
ErrorCode: -2147188685
line 2885 Error Details	:
lime 2929 ApiExceptionMessageName	: ImportSolutionError
ApiExceptionHttpStatusCode	: 500
0	: Microsoft.Crm.CrmException: The following solution cannot be imported: CALSingapore. Some dependencies are missing. The missing dependencies are : <MissingDependencies><MissingDependency><Required type="connectionreference" displayName="cr7d0_sharedregexmatchingip_cd463" solution="Active" id.connectionreferencelogicalname="cr7d0_sharedregexmatchingip_cd463" /><Dependent type="29" displayName="Step X - Email Matching - Mitanshu Garg" id="{c51507f9-350d-ef11-9f89-000d3a59ff6a}" /></MissingDependency></MissingDependencies> , ProductUpdatesOnly : False
   at Microsoft.Crm.Tools.ImportExportPublish.ImportSolutionHandler.ThrowIfDependenciesMissing(XmlNode solutionRoot, Boolean managed)
   at Microsoft.Crm.Tools.ImportExportPublish.ImportSolutionHandler.CheckImportPreConditions(Boolean checkMissingDependencies)
   at Microsoft.Crm.Tools.ImportExportPublish.ImportSolutionHandler.ImportItem()
line 2945 Error: Message: Solution manifest import: FAILURE: The following solution cannot be imported: CALSingapore. Some dependencies are missing. The missing dependencies are : <MissingDependencies><MissingDependency><Required type="connectionreference" displayName="cr7d0_sharedregexmatchingip_cd463" solution="Active" id.connectionreferencelogicalname="cr7d0_sharedregexmatchingip_cd463" /><Dependent type="29" displayName="Step X - Email Matching - Mitanshu Garg" id="{c51507f9-350d-ef11-9f89-000d3a59ff6a}" /></MissingDependency></MissingDependencies> , ProductUpdatesOnly : False
ErrorCode: -2147188685
Trace: 
Time: 26/06/2024 20:34:59
ErrorCode: -2147188685
line 2957 Error: Message: The following solution cannot be imported: CALSingapore. Some dependencies are missing. The missing dependencies are : <MissingDependencies><MissingDependency><Required type="connectionreference" displayName="cr7d0_sharedregexmatchingip_cd463" solution="Active" id.connectionreferencelogicalname="cr7d0_sharedregexmatchingip_cd463" /><Dependent type="29" displayName="Step X - Email Matching - Mitanshu Garg" id="{c51507f9-350d-ef11-9f89-000d3a59ff6a}" /></MissingDependency></MissingDependencies> , ProductUpdatesOnly : False
ErrorCode: -2147188707
Trace: 
Time: 26/06/2024 20:34:59
ErrorCode: -2147188707
DateUTC: 26/06/2024
TimeUTC: 20:35:00
HelpLink Url: Not Provided
Trace: Not Provided
line 2967 |EXCEPTION: System.ServiceModel.FaultException`1[Microsoft.Xrm.Sdk.OrganizationServiceFault]: Solution manifest import: FAILURE: The following solution cannot be imported: CALSingapore. Some dependencies are missing. The missing dependencies are : <MissingDependencies><MissingDependency><Required type="connectionreference" displayName="cr7d0_sharedregexmatchingip_cd463" solution="Active" id.connectionreferencelogicalname="cr7d0_sharedregexmatchingip_cd463" /><Dependent type="29" displayName="Step X - Email Matching - Mitanshu Garg" id="{c51507f9-350d-ef11-9f89-000d3a59ff6a}" /></MissingDependency></MissingDependencies> , ProductUpdatesOnly : False (Fault Detail is equal to Exception details: 
ErrorCode: 0x80048033
Message: Solution manifest import: FAILURE: The following solution cannot be imported: CALSingapore. Some dependencies are missing. The missing dependencies are : <MissingDependencies><MissingDependency><Required type="connectionreference" displayName="cr7d0_sharedregexmatchingip_cd463" solution="Active" id.connectionreferencelogicalname="cr7d0_sharedregexmatchingip_cd463" /><Dependent type="29" displayName="Step X - Email Matching - Mitanshu Garg" id="{c51507f9-350d-ef11-9f89-000d3a59ff6a}" /></MissingDependency></MissingDependencies> , ProductUpdatesOnly : False
TimeStamp: 2024-06-26T20:34:59.6250671Z
line 3180 Error: Message: Solution manifest import: FAILURE: The following solution cannot be imported: CALDaniel. Some dependencies are missing. The missing dependencies are : <MissingDependencies><MissingDependency><Required type="connectionreference" displayName="cr7d0_sharedregexmatchingip_cd463" solution="Active" id.connectionreferencelogicalname="cr7d0_sharedregexmatchingip_cd463" /><Dependent type="29" displayName="Step X - Email Matching - Mitanshu Garg" id="{c51507f9-350d-ef11-9f89-000d3a59ff6a}" /></MissingDependency></MissingDependencies> , ProductUpdatesOnly : False
ErrorCode: -2147188685

please I need your help to solve this using PAC to create a copy of BOTA to BOTB

I am available on MS teams daniel@educlaas.com and telephone +447902596815

there is no exists BOTB the solution is new and empty just like the video

https://www.youtube.com/watch?v=mgXiK3HeKGg

regards
Daniel

**Reply by haack42** [reply]
Hello,

The logs shows the workaround worked but now there is this issue from your log file:
_{"statuscode":500,"statusdescription":"Internal Server Error","errors":[{"code":10000,"message":"A record with the specified key values does not exist in connectionreference entity"}]}_ (the other errors predate the starting point of our conversation).

In the yaml there is: **connectionReferenceLogicalName: template.connectionreference.cr7d0_lifelongLearningCala.cr.fGwd7JH4**

Will have to run some tests with this setup to see what causes this "new" issue.

Regards,
Matthias

**Reply by dacostadaniel** [reply]
Hello

I much appreciate your ongoing help.

I have been at this all day over here. So this might help you further. I did edit the YAML to have "Inactive" to be "Active", like you suggested.

for  CALINT copilot in only manages

All

Custom (48)

System (2)

of the required for CAL copilot

All

Custom (333)

System (17)

[CALEXIST_yaml.txt](https://github.com/user-attachments/files/16102343/CALEXIST_yaml.txt)
[pac-log.txt](https://github.com/user-attachments/files/16102351/pac-log.txt)

if i got to power apps the solution titled CALINT only has the 50.

it works great  when I create a bot with nothing in it but the vanilla topics that are create when one creates a new bot.

regards
Daniel

**Reply by haack42** [reply]
Hello,

The last error on this pac log is: {"statuscode":500,"statusdescription":"Internal Server Error","errors":[{"code":10000,"message":"Entity 'Workflow' With Id = cae4dcb9-42f0-ee11-904b-000d3a339dfc Does Not Exist"}]}

I tried a similar copilot I created with a work flow, a connector, a filecomponent, and dialogcomponents. I could run "pac copilot extract-template ..." and then "pac copilot create ..." with the unmodified template and it worked. Note: I did deliberately not deactivate a topic as we know it is an issue.

Once before I had the same issue that you have with flows, can you check your topic with name "xg - CC - Email Matching ..." if this workflow id is existing? Note that this id is NOT the **flow id** shown in the URL in PowerAutomate when you open the flow but what is called the **process id** which you can find in the Dataverse table named **Process** in the column named "Process" (an id which does not change when solutions are imported into new environments).

Please note that I have seen another report about the issue you have here with the workflow but I was not able yet to reproduce it yet (except using explicitly an incorrect id). If you recall the steps that led to this please do share them.

Should the flow id cae4dcb9-42f0-ee11-904b-000d3a339dfc not exist in the table please try finding the correct one and replace it in the template with the process id (you may need to do this for each flow id), then try the import again.

Regards,
Matthias

**Reply by dacostadaniel** [reply]
Hello

So i have deleted all xg - CC - Email matching Topics 

see the steps and results. I have inserted the yaml and log file in the doc

[STEPS FOR MATTHIAS.docx](https://github.com/user-attachments/files/16132275/STEPS.FOR.MATTHIAS.docx)

this remains problem for us. I becoming anxious as we have to duplicate this CAL bot several times and not feasible to copy and paste topics each time manually.

regards
Daniel da Costa

**Reply by dacostadaniel** [reply]
Hello Sir

I am sorry to disturb; any way to progress this please?

regards
Daniel

**Reply by dacostadaniel** [reply]
Hello

By the way, the above using 1.32.8, the extract succeeded without error and the creare failed

I tried this with pac install latest ...Version 1.33.5. it cannot extract without errors. is quite disturbing.

regards
Daniel

**Reply by dacostadaniel** [reply]
Hello

Will I be receiving any more guidance on this issue please?

regards
Daniel

**Reply by haack42** [reply]
Hello,

Please use the version 1.32.8 which should not fail on the template extraction. The version 1.33.5 indeed has an issue for which a fix was provided today and the next release will contain it.

I tried a new copilot with a connector, extracted and created it... this worked. But it fails for your YAML, even when I use my working connector reference. I do not know why yet but I shared this with my colleagues.

Regards,
Matthias

**Reply by dacostadaniel** [reply]
Hello

I have doe what you requested; extract works but fails the create

Microsoft Windows [Version 10.0.19045.4651]
(c) Microsoft Corporation. All rights reserved.

C:\Users\Sevara Kenjaeva>pac use 1.32.8
Now using version 1.32.8 of the Microsoft PowerApps CLI.

C:\Users\Sevara Kenjaeva>pac copilot extract-template --environment 1e33b310-6f9d-e3ff-8941-a0862324b21e --bot a3261537-7772-4bea-808d-e712d5133511 --templateFileName CAL_2024_07_19_SOL_1_32.8.yaml -o
Connected as daniel@educlaas.com

Loaded 355 components for bot 'CAL' with id a3261537-7772-4bea-808d-e712d5133511. Primary language: English, supported languages:

CAL -> C:\Users\Sevara Kenjaeva\CAL_2024_07_19_SOL_1_32.8.yaml
[CAL_2024_07_19_SOL_1_32.8_yaml.txt](https://github.com/user-attachments/files/16311982/CAL_2024_07_19_SOL_1_32.8_yaml.txt)

C:\Users\Sevara Kenjaeva>pac virtual-agent create --schemaName cr7d0_CALINTERN4 --templateFileName CAL_2024_07_19_SOL_1_32.8.yaml --environment 1e33b310-6f9d-e3ff-8941-a0862324b21e --displayName CALINTERN4 --solution CALINTERN4_2024_07_19_SOL
Connected as daniel@educlaas.com

Loaded 355 components for bot 'CALINTERN4' with id f71902b5-fb94-48bf-973b-354415befcfa. Primary language: English, supported languages:
Sorry, the app encountered a non recoverable error and will need to terminate. The exception details have been captured and will be forwarded to the development team, if telemetry has been enabled. Exception Id: 65a06300-8a7a-42c5-ab43-3f2c61da59e6, Exception Type: Microsoft.Bot.Platform.Content.Exceptions.DataverseBadRequestException
The diagnostic logs can be found at: C:\Users\Sevara Kenjaeva\AppData\Local\Microsoft\PowerAppsCLI\Microsoft.PowerApps.CLI.1.32.8\tools\logs\pac-log.txt
[pac-log.txt](https://github.com/user-attachments/files/16312004/pac-log.txt)

I do not know why. Also i do not know how to interpret the log file. Mayb eyou can tell me so I can delete the topic that might be causing the issue. I looked in process but I do not know which column contains the identifier.

regards
Daniel

**Reply by dacostadaniel** [reply]
By the way, you can give the dev team my yaml and ask them to use it; if they cannot solve the issue then the extract and create solution is not much of a COE master piece as my yaml is real-world issue. I hope you can help with this.

regards Daniel

**Reply by dacostadaniel** [reply]
Hello Sir,

can I ask if there is any update on the issue I reported please? Is there a chance that I will receive a solution please?

regards
Daniel da Costa
